.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: Report an issue about using Custodian
+labels: [kind/bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what happened!
+      value: "A bug happened!"
```

Comment:
`"A bug happened!"` --> `"When I entered 2 + 2, I got the answer 6."`

This helps create a pattern between the id and description, therefore making it even easier for a user to understand what is being asked. Creates a prompt for user to write from/gives the user an idea of _how_ to answer the question.